.docker/standalone.Dockerfile

```diff
@@ -1,7 +1,7 @@
 # This Dockerfile should be run from the root FStar directory
 
 ARG ocaml_version=4.12
-FROM ocaml/opam:ubuntu-ocaml-$ocaml_version
+FROM ocaml/opam:ubuntu-21.10-ocaml-$ocaml_version
```

Comment:
Is the latest Ubuntu image not allowing proper installation of .NET 6.0.x?
From https://hub.docker.com/r/ocaml/opam 
> The ubuntu tag is always the latest release of Ubuntu.

Reply:
I think the answer is no. Ubuntu 22.04 is not listed on https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu

Reply:
Then, in that case, a comment should be added to that Dockerfile, referring to that page. (Such a comment should also explain why the URL to the Microsoft package repository is mentioning Ubuntu 21.04)

Reply:
I go to link, and notice that Ubuntu22.04 in mentioned here, so I update Docker file to target 22.02

Reason why I switch to explicit Ubuntu version, so it is easier to keep version of .NET custom feed and Ubuntu and Opam in sync. Before adding feed for .NET there was no need to maintain version of Ubuntu, since all commands was the same irrespectively to Ubuntu version, right now this is not a case, so better be explicit then have some strange issues down the road.

Reply:
I have bad news. Ubuntu 22.04 do not have `python` module, so updating Z3 become more pressing.

Reply:
It seems that Ubuntu 22.04 discontinued the `python` metapackage, to force users to explicitly choose between python2 and python3. So, with Ubuntu 22.04, you should be able to install `python2` instead.
Or, you could even try dropping that package altogether, since I'm not even sure we actually need it. I don't believe it is related to the Z3 version at all.

Reply:
https://ocaml.org/p/z3/4.8.5 list `conf-python-2-7` as dependency. When I fool around that's was only package which required Python 2.

Reply:
It seems that what was renamed was executable `python` to `python2` so I have to change bunch of scripts which I think outside of scope this PR. So I will stick with 21.10 version in the PR and either create PR later, or let somebody else update  to Ubuntu 22.
Separate question, do we need to maintain instructions for different version of Linux?